packages/components/pie-modal/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+## v0.1.0
+
+### Added
+
+- Basic modal component shell
+- README and LICENSE files
+- Initial TypeScript and Vite config
+- `declaration.d.ts` file for TypeScript
```

Comment:
was this generated manually? it doesn't confirm to our new format if so.

Reply:
Good point! 👍